### DISCUSSION TITLE
How to make an install look like app

### DISCUSSION THREAD

**Discussion by Ddady1** [zauberzeug/nicegui#2146] [Q&A]
### Question

Hi,
I'm trying to make an install look like script\app that will have a couple of windows that i can maneuver between them.
Something like the Tabs but with buttons of 'Next', 'Previous', 'Cancel' etc. instead.
I tried to do it with the Stepper but i found it very hard to find how to make the Stepper buttons to be conditional.

Any help will be appreciated.

**Comment by falkoschindler** [comment]
Maybe https://nicegui.io/documentation/tabs#switch_tabs_programmatically is what you're looking for?
